lib/chef-acceptance/test_suite.rb

```diff
@@ -9,12 +11,16 @@ class TestSuite
 
     attr_reader :acceptance_cookbook_dir
 
+    attr_reader :temp_dir
+
     attr_reader :recipes_dir
 
     attr_reader :chef_dir
 
     attr_reader :chef_config_file
 
+    attr_reader :dna_json_file
+
     attr_accessor :run_recipes
```

Comment:
We have too many directory & file paths here. I think the `attr_reader`pattern here is putting a lot of cognitive load when reading the code. We should probably have one at this level which points us to the base directory and construct the rest when they are needed or turn them into methods that derive the value from the base directory stored at the class level.